PhysicsTools/PatAlgos/python/patMuonMerger_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+mergedMuons = cms.EDProducer("PATMuonMerger",
+                             muons     = cms.InputTag("slimmedMuons"), 
+                             pfCandidates=cms.InputTag("packedPFCandidates"),
+                             otherTracks = cms.InputTag("lostTracks"),
+                             muonCut = cms.string("pt>15 && abs(eta)<2.4"),
```

Comment:
hi @sscruz @mandrenguyen are these cuts of relevance for [PR](https://github.com/cms-sw/cmssw/pull/30237) btw?

Reply:
@gkrintir Reading the PR description, I don't think there's any interference between this PR and #30237.  In the latter, we are modifying which muons get stored in the slimmedMuons collection.  Here we are enlarging the lostTracks collection, taking care to keep tracks in only one of packed PF cands, slimmedMuons and lostTracks IIUC.